docs/custom_ruleset.md

```diff
@@ -47,3 +47,79 @@ You can now click `Run Analysis` and your custom ruleset will be used.
 
 > [!WARNING]
 > You will need to make sure that the rules or rulesets have a target or source label that is selected. If the ruleset does not match a selected target or source, it will be filtered out and skipped.
+
+## Creating Rules and Rulesets
+
+### Creating Rules
+
+To create your own rules, first you need to know of a change that must happen to complete a migration. This could be because of a framework or library that you are using (either an internal one or external/open source one). For this section, I am going to create a rule and ruleset for an open source framework but the process should be similar for any custom rule.
+
+#### Finding A Rule
+
+To find a place for a rule, we are going to look at the release notes of the quarkus framework. If we see the [section](https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.18#kubernetes-client-fabric8) here. If we follow this to the migration guide for this dependency, we can see this [section](https://github.com/fabric8io/kubernetes-client/blob/main/doc/MIGRATION-v7.md#kubernetes-model-artifact-removed-).
+
+> The Maven artifact io.fabric8:kubernetes-model has been removed from the project and is no longer published.
+> This artifact was just an aggregator of some of the Kubernetes model artifacts and had no specific purpose. It is no longer published, the io.fabric8:kubernetes-client-api or io.fabric8:kubernetes-openshift-uberjar artifacts should be used instead.
+
+So for this, we will need to make sure that if the dependency `io.fabric8:kubernetes-model` is not used.
```

Comment:
Worth a 2nd read of this sentence, maybe a typo, maybe wanted to remove the word "if"

Reply:
updated let me know if the new wording makes more sense.

Reply:
Looks good